src/SonarScanner.MSBuild.PreProcessor/SonarScanner.MSBuild.PreProcessor.csproj

```diff
@@ -13,8 +13,8 @@
     <ProjectReference Include="..\SonarScanner.MSBuild.Common\SonarScanner.MSBuild.Common.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Google.Protobuf" Version="3.21.9" />
-    <PackageReference Include="Grpc.Tools" Version="2.50.0">
+    <PackageReference Include="Google.Protobuf" Version="3.21.12" />
+    <PackageReference Include="Grpc.Tools" Version="2.51.0">
```

Comment:
Wasn't really required for this PR but I've touched the code and took the chance to update the libraries.

Reply:
Hopefully, it will not bite us like when we updated the protobuf version for security.

Reply:
For security, we had the lib embedded in the plugin and served it to Visual Studio.  The scanner works independently and the difference in the version changed is small. I expect no surprise here. 

This is also used only in the caching context. The tests should cover this scenario.